### PR TITLE
tidb: remove fast and incremental analyze

### PR DESCRIPTION
**Pull request by hi-rustin**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Removed the fast analyze feature docs. We already have deprecated it and deleted it.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md
Comment (Oreoxmt) [inline basic-features.md:1]: Please discard changes in this file and I have updated the basic-features for v7.5 in https://github.com/pingcap/docs-cn/pull/15273/commits/e184df166f91715639e0c49ddb9b68383e42dc57

File: system-variables.md [hunk 1]
```diff
@@ -1758,20 +1758,6 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 默认值：`OFF`
 - 该变量指定 TiDB 是否收集扩展统计信息来指导优化器。
 
-### `tidb_enable_fast_analyze`
```
Comment (Oreoxmt) [inline system-variables.md:1761]: Does the item `tidb_enable_fast_analyze` in the output of `SHOW GLOBAL VARIABLES LIKE 'tidb%';` need to be deleted?

https://github.com/pingcap/docs-cn/blob/2c92925651c1b7e137fe4f75ee8beec8026dcbca/sql-statements/sql-statement-show-variables.md?plain=1#L65

Comment (hi-rustin) [reply system-variables.md:1761]: Updated.

File: basic-features.md
Comment (hi-rustin) [reply basic-features.md:1]: Reverted.

File: sql-statements/sql-statement-analyze-table.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-analyze-table/','/docs-cn/d
 
 当 TiDB 逐渐发现这些统计数据与预估不一致时，也会自动更新其统计数据。
 
-目前 TiDB 收集统计信息分为全量收集和增量收集两种方式，分别通过 `ANALYZE TABLE` 和 `ANALYZE INCREMENTAL TABLE` 语句来实现。关于这两种语句的详细使用方式，可参考[统计信息简介](/statistics.md)。
+目前 TiDB 收集统计信息为全量收集，分别通过 `ANALYZE TABLE` 语句来实现。关于该语句的详细使用方式，可参考[统计信息简介](/statistics.md)。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-analyze-table.md:13]:
```suggestion
目前 TiDB 收集统计信息为全量收集，通过 `ANALYZE TABLE` 语句来实现。关于该语句的详细使用方式，可参考[统计信息简介](/statistics.md)。
```

File: statistics.md [hunk 3]
```diff
@@ -84,12 +84,7 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
 > **注意：**
 >
 > 在 TiDB 中执行 `ANALYZE TABLE` 语句比在 MySQL 或 InnoDB 中耗时更长。InnoDB 采样的只是少量页面，但 TiDB 会完全重构一系列统计信息。适用于 MySQL 的脚本会误以为执行 `ANALYZE TABLE` 耗时较短。
->
-> 如需更快的分析速度，可将 `tidb_enable_fast_analyze` 设置为 `1` 来打开快速分析功能。该参数的默认值为 `0`。
->
-> 快速分析功能开启后，TiDB 会随机采样约 10000 行的数据来构建统计信息。因此在数据分布不均匀或者数据量比较少的情况下，统计信息的准确度会比较差。可能导致执行计划不优，比如选错索引。如果可以接受普通 `ANALYZE` 语句的执行时间，则推荐关闭快速分析功能。
->
-> `tidb_enable_fast_analyze` 为实验性功能，目前与 `tidb_analyze_version=2` 的统计信息**不完全匹配**。因此开启 `tidb_enable_fast_analyze` 时需要将 `tidb_analyze_version` 的值设置为 `1`。
+> TiDB 的实验特性快速分析和增量分析已经在 v7.5.0 版本中移除。
```
Comment (Oreoxmt) [inline statistics.md:87]:
```suggestion
> 从 v7.5.0 开始，统计信息[快速分析 (`tidb_enable_fast_analyze`)](https://docs.pingcap.com/zh/tidb/v7.4/system-variables#tidb_enable_fast_analyze) 和[增量收集](https://docs.pingcap.com/zh/tidb/v7.4/statistics#增量收集)废弃。
```

File: statistics.md [hunk 4]
```diff
@@ -84,12 +84,7 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
 
 #### 全量收集
```
Comment (Oreoxmt) [inline statistics.md:89]:
```suggestion
```

File: statistics.md [hunk 5]
```diff
@@ -84,12 +84,7 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
 > **注意：**
```
Comment (Oreoxmt) [inline statistics.md:84]: L82:

```
目前 TiDB 收集统计信息为全量收集，通过 `ANALYZE TABLE` 语句来实现。
```

Comment (hi-rustin) [reply statistics.md:84]: 没太明白你的意思

Comment (Oreoxmt) [reply statistics.md:84]: 因为下面删除了“全量收集”的标题，所以可以在“手动收集”开头说明一下收集统计信息是全量收集
https://github.com/pingcap/docs-cn/blob/4c5ec03bf5836fd9e78fa5fc6b09d72795123927/statistics.md?plain=1#L80-L82

File: statistics.md [hunk 6]
```diff
@@ -84,14 +84,7 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
 >
 > 在 TiDB 中执行 `ANALYZE TABLE` 语句比在 MySQL 或 InnoDB 中耗时更长。InnoDB 采样的只是少量页面，但 TiDB 会完全重构一系列统计信息。适用于 MySQL 的脚本会误以为执行 `ANALYZE TABLE` 耗时较短。
```
Comment (Oreoxmt) [inline statistics.md:86]:
```suggestion
> - 在 TiDB 中执行 `ANALYZE TABLE` 语句比在 MySQL 或 InnoDB 中耗时更长。InnoDB 采样的只是少量页面，但 TiDB 会完全重构一系列统计信息。适用于 MySQL 的脚本会误以为执行 `ANALYZE TABLE` 耗时较短。
```

File: statistics.md [hunk 7]
```diff
@@ -84,14 +84,7 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
->
-> 如需更快的分析速度，可将 `tidb_enable_fast_analyze` 设置为 `1` 来打开快速分析功能。该参数的默认值为 `0`。
->
-> 快速分析功能开启后，TiDB 会随机采样约 10000 行的数据来构建统计信息。因此在数据分布不均匀或者数据量比较少的情况下，统计信息的准确度会比较差。可能导致执行计划不优，比如选错索引。如果可以接受普通 `ANALYZE` 语句的执行时间，则推荐关闭快速分析功能。
->
-> `tidb_enable_fast_analyze` 为实验性功能，目前与 `tidb_analyze_version=2` 的统计信息**不完全匹配**。因此开启 `tidb_enable_fast_analyze` 时需要将 `tidb_analyze_version` 的值设置为 `1`。
-
-#### 全量收集
+> 从 v7.5.0 开始，统计信息[快速分析 (`tidb_enable_fast_analyze`)](https://docs.pingcap.com/zh/tidb/v7.4/system-variables#tidb_enable_fast_analyze) 和[增量收集](https://docs.pingcap.com/zh/tidb/v7.4/statistics#增量收集)废弃。
```
Comment (Oreoxmt) [inline statistics.md:87]:
```suggestion
> - 从 v7.5.0 开始，统计信息[快速分析 (`tidb_enable_fast_analyze`)](https://docs.pingcap.com/zh/tidb/v7.4/system-variables#tidb_enable_fast_analyze) 和[增量收集](https://docs.pingcap.com/zh/tidb/v7.4/statistics#增量收集)废弃。
```

File: statistics.md (hunk 5 above)
Comment (hi-rustin) [reply statistics.md:84]: 更新了。

**Review by Oreoxmt — APPROVED** [review]

**Review by songrijie — APPROVED** [review]
LGTM

File: sql-statements/sql-statement-analyze-table.md [hunk 8]
```diff
@@ -129,12 +129,10 @@ EXPLAIN SELECT * FROM t1 WHERE c1 = 3;
 
 * `ANALYZE TABLE` 在语法上与 MySQL 类似。但 `ANALYZE TABLE` 在 TiDB 上的执行时间可能长得多，因为它的内部运行方式不同。
```
Comment (ran-huang) [inline sql-statements/sql-statement-analyze-table.md:130]:
```suggestion
`ANALYZE TABLE` 在语法上与 MySQL 类似。但 `ANALYZE TABLE` 在 TiDB 上的执行时间可能长得多，因为它的内部运行方式不同。
```

**Review by ran-huang — APPROVED** [review]

File: system-variables.md [hunk 9]
```diff
@@ -1758,20 +1758,6 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 默认值：`OFF`
 - 该变量指定 TiDB 是否收集扩展统计信息来指导优化器。
```
Comment (Oreoxmt) [inline system-variables.md:1760]:
```suggestion

### `tidb_enable_fast_analyze`

> **警告：**
>
> 从 v7.5.0 开始，该变量被废弃。

- 作用域：SESSION | GLOBAL
- 是否持久化到集群：是
- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
- 类型：布尔型
- 默认值：`OFF`
- 这个变量用来控制是否启用统计信息快速分析功能。默认值 0 表示不开启。
- 快速分析功能开启后，TiDB 会随机采样约 10000 行的数据来构建统计信息。因此在数据分布不均匀或者数据量比较少的情况下，统计信息的准确度会比较低。这可能导致执行计划不优，比如选错索引。如果可以接受普通 `ANALYZE` 语句的执行时间，则推荐关闭快速分析功能。
```

File: system-variables.md [hunk 10]
```diff
@@ -1771,7 +1771,6 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 默认值：`OFF`
 - 这个变量用来控制是否启用统计信息快速分析功能。默认值 0 表示不开启。
 - 快速分析功能开启后，TiDB 会随机采样约 10000 行的数据来构建统计信息。因此在数据分布不均匀或者数据量比较少的情况下，统计信息的准确度会比较低。这可能导致执行计划不优，比如选错索引。如果可以接受普通 `ANALYZE` 语句的执行时间，则推荐关闭快速分析功能。
-
 ### `tidb_enable_fast_table_check` <span class="version-mark">从 v7.2.0 版本开始引入</span>
```
Comment (hi-rustin) [inline system-variables.md:1774]:
```suggestion

### `tidb_enable_fast_table_check` <span class="version-mark">从 v7.2.0 版本开始引入</span>
```

**Pull request merged**